css/avatar.css

```diff
@@ -0,0 +1,37 @@
+.avatar {
+    position: relative;
+    width: 100px;
+    margin: 100px auto;
+}
+
+.face img {
+    height: 100px;
+    position: absolute;
+    left: 110px;
+    z-index: 5;
+    transform: rotate(280deg);
+}
+
+.car img {
+    height: 100px;
+}
+
+.left {
+     transform: rotate(180deg);
+ }
+
+.right {
+    transform: rotate(0deg);
+}
+
+.up {
+    transform: rotate(270deg);
+}
+
+.down {
+    transform: rotate(90deg);
+}
+
+#carColor {
+    fill: gold;
```

Comment:
hex code here (upper case)?

Reply:
removed as setting colour in the svg file